packages/synapse-react-client/src/components/widgets/RadioGroup.tsx

```diff
@@ -36,28 +36,29 @@ export type RadioOptionProps<T extends string | boolean | number = string> = {
   value: T
   style?: React.CSSProperties
   onChange: (value: T) => void
-  disabled?: boolean
-  checked: boolean
-}
+} & Omit<
+  React.HTMLProps<HTMLInputElement>,
+  'label' | 'value' | 'style' | 'onChange'
+>
```

Comment:
Just changed the prop signature to allow passing any valid prop to the `input` component. This let me pass an `aria-label` prop in `ChallengeTeamTable`